### PR TITLE
Reward - managing kudos - 1 screenshot missing

### PR DESCRIPTION
**Pull request by samehaidi**

### CODE REVIEW

File: docs/Administration.rst [hunk 1]
```diff
@@ -4616,6 +4619,31 @@ The wallet does not contain enough ether to make transactions:  For these wallet
...
 
 
+.. _ManagingKudos:
+
+===============
+Managing Kudos
+===============
+
+You can manage the use of kudos in the platform from "Kudos settings" page. 
+To access to this interface, you have to click on "Administration" menu from the top navigation, then "Reward" - "Kudos" from the drop-down menu.
```
Comment (plamarque) [inline docs/Administration.rst]:
```suggestion
To access to this interface, click on "Administration" menu from the top navigation, then "Reward" > "Kudos" from the drop-down menu.
```

File: docs/Administration.rst [hunk 2]
```diff
@@ -4616,6 +4619,31 @@ The wallet does not contain enough ether to make transactions:  For these wallet
...
+
+|image333|
+
+You can set the number of the allocated kudos for users to use per period of time (week, month, quarter or year).
```
Comment (plamarque) [inline docs/Administration.rst]:
```suggestion
You can define how many kudos users can send  per period of time (week, month, quarter or year). 
```

File: docs/Administration.rst [hunk 3]
```diff
@@ -4616,6 +4619,31 @@ The wallet does not contain enough ether to make transactions:  For these wallet
...
+Managing Kudos
+===============
+
+You can manage the use of kudos in the platform from "Kudos settings" page. 
+To access to this interface, you have to click on "Administration" menu from the top navigation, then "Reward" - "Kudos" from the drop-down menu.
+
+|image333|
+
+You can set the number of the allocated kudos for users to use per period of time (week, month, quarter or year). 
+You can also restrict the use of Kudos to the members of a single space. They will be allowed to send and recieve Kudos even from other space's activities. In this case, both sender and reciever must be in the specified space.
```
Comment (plamarque) [inline docs/Administration.rst]:
```suggestion
You can also restrict the use of Kudos to the members of a single space. Here space membership is simply used to indicate a group permission. Authorized users will be able to send and receive kudos from any other space.
```

File: docs/Administration.rst [hunk 4]
```diff
@@ -4616,6 +4619,31 @@ The wallet does not contain enough ether to make transactions:  For these wallet
...
+
+|image334|
+
+You can access to the list of the sent kudos from the tab "Kudos List" of the same interface. The information of the sender and reciever are available.
```
Comment (plamarque) [inline docs/Administration.rst]:
```suggestion
You can review the list of kudos received and sent from the tab "Kudos List" of the same interface. The information of the sender and receiver are available.
```

File: managing rewards kudos ..etc [hunk 5]
```diff
@@ -0,0 +1,86 @@
+.. _ManagingKudos:
+
+===============
```
Comment (plamarque) [inline managing rewards kudos ..etc]: @samehaidi this hould be removed. I think it's duplicatedfrom Administration.rst

File: managing rewards kudos ..etc [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+
+.. _ManagingRewards:
+
+================
+Managing Rewards
+================
+
+You can manage the rewards sending from "Reward administration" page. 
+To access to this interface, you have to click on "Administration" menu from the top navigation, then "Reward" - "Reward" from the drop-down menu.
```
Comment (plamarque) [inline managing rewards kudos ..etc]:
```suggestion
To access to this interface,  click on "Administration" menu from the top navigation, then "Reward" > "Reward" from the drop-down menu.
```

File: managing rewards kudos ..etc [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+Finally, when everything is setup correctly in both configuration and pools tabs, you can send rewards from the first tab.
+
+
+Configuration
+~~~~~~~~~~~~~
+
+Configuration is the third tab of Reward interface. 
+This tab allows you to manage the reward's sending, the periodicity and how rewards will be calculated for users at the end of each period.
+
+|image|
```
Comment (plamarque) [inline managing rewards kudos ..etc]: image missing

File: managing rewards kudos ..etc [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+Since the use of the platform may not be the same from one team to another, and in order to ensure a fair distribution of rewards, you can divide users by pools according to their teams. 
+This will put the members of the same team in competition together but not with all the users of the platform.
+Finally, when everything is setup correctly in both configuration and pools tabs, you can send rewards from the first tab.
+
+
+Configuration
+~~~~~~~~~~~~~
+
+Configuration is the third tab of Reward interface. 
+This tab allows you to manage the reward's sending, the periodicity and how rewards will be calculated for users at the end of each period.
```
Comment (plamarque) [inline managing rewards kudos ..etc]:
```suggestion
This tab lets you to manage the rewards sending, the periodicity and how rewards will be calculated for users at the end of each period.
```

File: managing rewards kudos ..etc [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+
+|image|
+
+You have to choose the periodicity of sending rewards. The possible values are : week, month, quarter, semester or year.
```
Comment (plamarque) [inline managing rewards kudos ..etc]:
```suggestion
Choose the periodicity for sending rewards. The possible values are : week, month, quarter, semester or year.
```

File: managing rewards kudos ..etc [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+The interface is divided into two blocs : a bloc to setup the rewards for recieved kudos and a bloc to setup the rewards for gamification points.
```
Comment (plamarque) [inline managing rewards kudos ..etc]:
```suggestion
The interface is divided into two blocks : a block to setup the rewards for kudos and a block to setup the rewards for gamification points.
```

File: managing rewards kudos ..etc [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+For each bloc you will have to specify if there is a minimal threshold to reward users. The default value is "0", which means that the users are rewarded automatically if they recieve kudos or earn any gamification points.
```
Comment (plamarque) [inline managing rewards kudos ..etc]:
```suggestion
For each bloc you can specify a minimal threshold to reward users. The default value is "0", which means that the users are rewarded automatically if they receive kudos or earn any gamification points.
```

File: managing rewards kudos ..etc [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+
+You have to select the checkbox "Use pools" if you want to use the pools in the rewards calculation and this for kudos and gamification.
```
Comment (plamarque) [inline managing rewards kudos ..etc]:
```suggestion
Select the checkbox "Use pools" if you want to use the pools in the rewards calculation and this for kudos and gamification.
```

File: managing rewards kudos ..etc [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+
+You have three choices to set the reward budget for each both blocs :
```
Comment (plamarque) [inline managing rewards kudos ..etc]:
```suggestion
For each case, you have three ways for controlling your rewarding budget :
```

File: managing rewards kudos ..etc [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+
+- **By a fixed budget:** This option allows you to set a total amount of token for all users rewards. This amount will be divided between the different users. The calculation will be done according to the chosen mode (use or not of pools).
```
Comment (plamarque) [inline managing rewards kudos ..etc]:
```suggestion
- **By a fixed budget:** This option allows you to set a total amount of tokens for all users rewards. This amount will be divided between the different users. The calculation will be done according to the chosen mode (use pools or not). 
```

File: managing rewards kudos ..etc [hunk 15]
```diff
@@ -0,0 +1,86 @@
...
+You have to choose the periodicity of sending rewards. The possible values are : week, month, quarter, semester or year.
+The interface is divided into two blocs : a bloc to setup the rewards for recieved kudos and a bloc to setup the rewards for gamification points.
+For each bloc you will have to specify if there is a minimal threshold to reward users. The default value is "0", which means that the users are rewarded automatically if they recieve kudos or earn any gamification points.
+
+You have to select the checkbox "Use pools" if you want to use the pools in the rewards calculation and this for kudos and gamification.
+
+You have three choices to set the reward budget for each both blocs :
+
+- **By a fixed budget:** This option allows you to set a total amount of token for all users rewards. This amount will be divided between the different users. The calculation will be done according to the chosen mode (use or not of pools). 
+- **By a fixed budget per eligible member:** This option alows you to fix a budget to be divided between the eligible members.  The calculation will be done according to the chosen mode (use or not of pools).
```
Comment (plamarque) [inline managing rewards kudos ..etc]:
```suggestion
- **By a fixed budget per eligible member:** This mode lets you set a dynamic budget sized according to the number of active (eligible) users.  The calculation will be done according to the chosen mode (use of pools or not).
```

File: managing rewards kudos ..etc [hunk 16]
```diff
@@ -0,0 +1,86 @@
...
+- **By a fixed budget per point:** This option alows you to fix a budget per point (for each recieved kudos or earned gamification points).
```
Comment (plamarque) [inline managing rewards kudos ..etc]:
```suggestion
- **By a fixed budget per point:** This mode lets you define a budget per so-called "point" (for each kudos or gamification received).
```

**Review by plamarque — CHANGES_REQUESTED** [review]
first pass typos and simplifications

File: managing rewards kudos ..etc (hunk 5 above)
Comment (samehaidi) [reply managing rewards kudos ..etc]: as explained, this file .rst is not meant to be in this PR

File: docs/Administration.rst [hunk 17]
```diff
@@ -4616,6 +4619,31 @@ The wallet does not contain enough ether to make transactions:  For these wallet
...
 
 
+.. _ManagingKudos:
+
+===============
+Managing Kudos
+===============
+
+You can manage the use of kudos in the platform from "Kudos settings" page. 
+To access to this interface, click on "Administration" menu from the top navigation, then "Reward" > "Kudos" from the drop-down menu.
```
Comment (plamarque) [inline docs/Administration.rst]:
```suggestion
To access this interface, click on "Administration" menu from the top navigation, then "Reward" > "Kudos" from the drop-down menu.
```

File: docs/Administration.rst [hunk 18]
```diff
@@ -4616,6 +4619,31 @@ The wallet does not contain enough ether to make transactions:  For these wallet
...
+
+|image333|
+
+You can define how many kudos users can send  per period of time (week, month, quarter or year). 
+You can also restrict the use of Kudos to the members of a single space. Here space membership is simply used to indicate a group permission. Authorized users will be able to send and receive kudos from any other space.
```
Comment (plamarque) [inline docs/Administration.rst]:
```suggestion
You can also restrict the use of kudos to the members of a single space. Here, the space membership is simply used as a group permission. Authorized users will be able to send and receive kudos from any other space.
```

**Review by plamarque — CHANGES_REQUESTED** [review]
images missing and duplicate text to fix. Also a couple of smaller edits suggested.

**Comment by samehaidi** [comment]
Extra file deleted, suggestions commited

**Review by plamarque — APPROVED** [review]
LGTM

**Pull request merged**